notesapi/utils.py

```diff
@@ -0,0 +1,124 @@
+"""
+Paginator and Serializers.
+"""
+
+import json
+
+from rest_framework import pagination
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from notesapi.v1.models import Note
+
+
+class NotesPaginator(pagination.PageNumberPagination):
+    """
+    Student Notes Paginator.
+    """
+    page_size = 10
+    page_size_query_param = "page_size"
+
+    def get_paginated_response(self, data):
+        """
+        Annotate the response with pagination information.
+        """
+        return Response({
+            'current': self.page.number,
+            'next': self.get_next_link(),
+            'previous': self.get_previous_link(),
+            'count': self.page.paginator.count,
+            'num_pages': self.page.paginator.num_pages,
+            'results': data
+        })
+
+
+class NotesSerializer(serializers.ModelSerializer):
+    """
+    Student Notes Model Serializer.
+    """
+    class Meta(object):
+        """
+        Model Serializer Meta Class.
+        """
+        model = Note
+        exclude = ('user_id',)
+
+    id = serializers.SerializerMethodField()
```

Comment:
Don't we have this field in model. Can we use that field as it, if not, we can make field like this --> `serializers.Field(source='SourceFiledName')`